### ISSUE TITLE
TabBarController - bug on selecting tabs programmatically

### ISSUE THREAD

**Issue by melcug** [CosmicMind/Material#833]
![simulator screen shot aug 10 2017 13 10 03 1](https://user-images.githubusercontent.com/8708178/29166569-9f639e5a-7dcf-11e7-9bd5-c9794bf1576d.jpg)

I use following code the select second tab from TabBar:
`
navigationDrawerController?.rootViewController.tabsController?.tabBar.select(at: 1)
`

As you can see, the result is incomplete: the tab is selected, but associated view is not shown and previous view is still visible.

Regards,
Gogu.

**Comment by danieldahan** [comment]
You can write this now with the latest Material 2.9.* version. 

```swift
tabsController?.tabBar.select(at: 1)
```

and the select feature has not yet been implemented. I will make that update most likely tomorrow. Thank you!

**Comment by danieldahan** [comment]
You can now find a fix for this in [Material 2.10.1](https://github.com/CosmicMind/Material/releases/tag/2.10.1).

Use:

```swift
tabsController?.select(at: 1)
```

Thank you!
